pkg/controllers/vdb/offlineupgrade_reconciler.go

```diff
@@ -287,6 +291,12 @@ func (o *OfflineUpgradeReconciler) postRestartingClusterMsg(ctx context.Context)
 	return o.postNextStatusMsg(ctx, ClusterRestartOfflineMsgIndex)
 }
 
+// postInstallPackagesMsg will update the status message to indicate the
+// default packages are installed after the first restart of all subcluster(s)
+func (o *OfflineUpgradeReconciler) postInstallPackagesMsg(ctx context.Context) (ctrl.Result, error) {
+	return o.postNextStatusMsg(ctx, ClusterRestartOfflineMsgIndex)
```

Comment:
I don't think using `ClusterRestartOfflineMsgIndex` will work. I think it needs to be a new index value.

Reply:
will do